dev/preview/infrastructure/harvester/svc.tf

```diff
@@ -56,7 +56,7 @@ resource "kubernetes_service" "harvester-svc" {
       name        = "https"
       protocol    = "TCP"
       port        = 443
-      target_port = 4430
```

Comment:
Was this a bug in the original tf code or why did this port need to change? ☺️

Reply:
Ah yeah, forgot to mention that one - it was the wrong target port.